lib/policies/load-balancing.js

```diff
@@ -294,15 +297,29 @@ TokenAwarePolicy.prototype.newQueryPlan = function (keyspace, queryOptions, call
  */
 function TokenAwareIterator(keyspace, queryOptions, replicas, childPolicy) {
```

Comment:
Good arguments.
Additionally, the driver exposes the `client.getReplicas()` that users does to retrieve the replicas and the ordering haven't changed (closest token and the following replicas) but in the case of `TokenAwarePolicy` I think that providing the closest replica first is a bug and we should always distribute the load.